base/views.py

```diff
@@ -55,13 +55,21 @@ def index(request):
 
     activity_list = Activity.objects.filter(date__gte=date.today()).order_by('date')
 
+    protect_personal_data = (
+        settings.YNH_INTEGRATION_ENABLED
+        and
+        request.user.is_anonymous
```

Comment:
On peut pas dire plutôt "not have permission" ?

Reply:
On utilise pas les permissions (ça serait idéal mais c'est du code en plus).